### PR TITLE
CADC-10275

### PR DESCRIPTION
**Pull request by SharonGoliath**

### CODE REVIEW

File: dao2caom2/telescopes.py [hunk 1]
```diff
@@ -166,15 +166,21 @@ def update(self, observation, file_info, clients):
                         cc.undo_astropy_cdfix_call(chunk, time_delta)
 
                         if self._storage_name.file_name.startswith('d'):
-                            if plane.data_product_type == DataProductType.SPECTRUM:
+                            if (
```
Comment (andamian) [inline dao2caom2/telescopes.py]: I'm thinking we should move to 120-wide lines to increase readability in cases like this.

File: dao2caom2/telescopes.py [hunk 2]
```diff
@@ -698,11 +698,26 @@ def _get_position(self, ext):
...
                 fk5 = FK5(equinox=equinox)
-                coord = SkyCoord(
-                    f'{ra} {dec}', unit=(u.hourangle, u.deg), frame=fk5
-                )
+                try:
+                    coord = SkyCoord(
+                        f'{ra} {dec}', unit=(u.hourangle, u.deg), frame=fk5
+                    )
+                except ValueError as e:
+                    if '-90 deg <= angle <= 90 deg' in str(e):
```
Comment (andamian) [inline dao2caom2/telescopes.py:728]: The comment refers to `DEC > +90` but the code does that for `DEC < -90` as well. Is that intentional?

**Review by andamian — APPROVED** [review]
Reviewed the last 7 commits.

File: dao2caom2/telescopes.py (hunk 1 above)
Comment (SharonGoliath) [reply dao2caom2/telescopes.py]: Two questions:
- change all repositories?
- will there be one commit with all the line length changes in it, to master?

File: dao2caom2/telescopes.py (hunk 2 above)
Comment (SharonGoliath) [reply dao2caom2/telescopes.py:728]: Nice catch.

**Pull request merged**